contributors/triage/guideline.md

```diff
@@ -0,0 +1,62 @@
+
+## Find the right SIG(s)
+Components are divided among [Special Interest Groups (SIGs)](https://github.com/kubernetes/community/blob/master/sig-list.md). Find a proper SIG for the ownership of the issue using the bot:
+
+* Typing `/sig network` in a comment should add the sig/network label, for example.
+* Multiword SIGs use dashes, for example `/sig cluster-lifecycle`.
+
+## Validate if the issue is real
+
+Validate if the problem is real by reproducing it. If reproducible, move to the next step of defining priority. You may need to contact the issue reporter in the following cases:
+* Do a quick duplicate search to see if the issue has been reported already. If a duplicate is found, let the issue reporter know it by marking it duplicate.
+* If you can not reproduce the issue, contact the issue reporter with your findings and close the issue if both the parties agree that it could not be reproduced.
+* If the issue is non-trivial to reproduce, work with issue reporter and let SIG know of your findings.
+* If you do not get a response in 20 days, then close the issue with appropriate comment.
+
+## Define priority
+After you validate the issue, add a comment of what you think of priority of fixing issue from priority labels provided below. The owner from appropriate SIG will correct the label if needed.
+* `priority/critical-urgent` or `priority/failing-test` - Critical. Fix it now.
+* `priority/important-soon` - High priority. Fix it as soon as possible.
+* `priority/important-longterm` - High priority over the long duration.
+* `priority/backlog` - Medium priority.
+* `priority/awaiting-more-evidence` - Low priority.
+
+To read priorities in detail, please refer to the [Priorities](https://github.com/kubernetes/community/blob/master/contributors/devel/issues.md#priorities).
+
+## Set ownership
+
+All issues should have an owner. If you are not sure of an owner, defer to the SIG label only. If you feel the issue should warrant a notification,you can ping a team with an @ mention, in this format, `@kubernetes/sig-<group-name>-<group-suffix>`. Here the `<group-suffix>` can be one of `bugs, feature-requests, pr-reviews, test-failures, proposals`.
+For example, `@kubernetes/sig-cluster-lifecycle-bugs, can you have a look at this?`
+
+If you think you can fix the issue and you are an issue reporter or a component organization member, assign it to yourself with just `/assign`. If you can not self-assign, leave a comment that you are willing to work on it and work on creating a PR.
+
+## Poke issue owner if PR is not created for it in 30 days
```

Comment:
@bgrant0607 For now, in new commit, didn't make much changes here. Totally agree with your comment but if there is a reasonable comment in the issue that it will take longer, then trigger won't bug SIG after initial touch base. If we need to make changes to text, I am good with it but for now created new commit with most feedback. Thanks!